### PR TITLE
chore: improving GPU QOS with retries and smaller sku selection

### PR DESCRIPTION
**Pull request by Bryce-Soghigian**
Fixes # 

**Description**
When we use just the `n` series selector, we go through many instance types, we hit no quota for those gpu types a couple of times before successful provisioning for the "Standard_NC4as_T4_v3"  SKU. This just changes it to speed along this process. 
 
**How was this change tested?**
`/test`

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [x] No

**Release Note**

```release-note

```

### CODE REVIEW

**Review by tallaxes — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Comment by Bryce-Soghigian** [comment]
going to run additional /test throughout the day to get some QOS on GPU

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

**Review by Bryce-Soghigian — COMMENTED** [review]
/test

File: test/suites/gpu/suite_test.go [hunk 1]
```diff
@@ -148,7 +153,8 @@ var _ = Describe("GPU", func() {
 		}
 		env.ExpectCreated(nodeClass, nodePool, deployment, devicePlugin)
 		// This test exercises the full lifecycle of the GPU Node, and validates it can successfully schedule GPU Resources.
-		env.EventuallyExpectHealthyPodCountWithTimeout(time.Minute*15, labels.SelectorFromSet(deployment.Spec.Selector.MatchLabels), int(*deployment.Spec.Replicas))
+		// the Time.Minute 45 gives the test 3 tries to provision and bootstrap the node successfully
```
Comment (charliedmcb) [inline test/suites/gpu/suite_test.go:156]: Add a TODO comment to try and improve this time requirement.

I don't have any problem with going up to 45 minutes if its improving the test pass rate for now, as I think that is more important currently. However, another side of QoS is complete runtime of the test, which is something we'll want to target to reduce as well

**Review by charliedmcb — CHANGES_REQUESTED** [review]
Assuming you've gotten a confirmed improved pass rate this is looking like a good change.

Please do update the PR description with a comment on the reasoning for the increased timeout as well, plus add links to the actual testing done, or at least more details of what you did as I believe it went beyond just one run of `/test`?

**Pull request closed without merging**